docs/examples/ratify-verify-azure-cmd.md

```diff
@@ -1,296 +1,347 @@
+
+### Set up Workload Identity with AKS, ACR and AKV:
+The official steps for setting up Workload Identity on AKS can be found here.
+
+1. Configure environment variables.
+
+```
+export IDENTITY_NAME=<Identity Name>
+export GROUP_NAME=<Azure Resource Group Name>
+export SUBSCRIPTION_ID=<Azure Subscription ID>
+export TENANT_ID=<Azure Tenant ID>
+export ACR_NAME=<Azure Container Registry Name>
+export AKS_NAME=<Azure Kubernetes Service Name>
+export KEYVAULT_NAME=<Azure Key Vault Name>
+export RATIFY_NAMESPACE=<Namespace where Ratify deployed, defaults to "gatekeeper-system">
+export CERT_DIR=<Directory storing TLS certs>
+export NOTARY_PERM_NAME=<Name of cerficicate file uploaded to Key Vault>
+```
+2. Create a Workload Federated Identity.
+
+```
+az identity create --name "${IDENTITY_NAME}" --resource-group "${GROUP_NAME}" --location "${LOCATION}" --subscription "${SUBSCRIPTION_ID}"
+
+export IDENTITY_OBJECT_ID="$(az identity show --name "${IDENTITY_NAME}" --resource-group "${GROUP_NAME}" --query 'principalId' -otsv)"
+export IDENTITY_CLIENT_ID=$(az identity show --name ${IDENTITY_NAME} --resource-group ${GROUP_NAME} --query 'clientId' -o tsv)
+```
+
+3. Create ACR and enable `AcrPull` role to the workload identity.
+
+```
+az acr create --name "${ACR_NAME}" \
+  --resource-group "${GROUP_NAME}" \
+  --sku Standard
+```
+
+4. Configure user-assigend managed identity
+
+```
+az role assignment create \
+  --assignee-object-id ${IDENTITY_OBJECT_ID} \
+  --role acrpull \
+  --scope subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${GROUP_NAME}/providers/Microsoft.ContainerRegistry/registries/${ACR_NAME}
+```
+
+5. Create OIDC enabled AKS cluster by following the steps below:
+
+```
+# Install the aks-preview extension
+az extension add --name aks-preview
+
+# Register the 'EnableWorkloadIdentityPreview' feature flag
+az feature register --namespace "Microsoft.ContainerService" --name "EnableWorkloadIdentityPreview"
+az provider register --namespace Microsoft.ContainerService
+
+az aks create \
+    --resource-group "${GROUP_NAME}" \
+    --name "${AKS_NAME}" \
+    --node-vm-size Standard_DS3_v2 \
+    --node-count 1 \
+    --generate-ssh-keys \
+    --enable-workload-identity \
+    --attach-acr ${ACR_NAME} \
+    --enable-oidc-issuer
+
+# Connect to the AKS cluster:
+az aks get-credentials --resource-group ${GROUP_NAME} --name ${AKS_NAME}
+
+export AKS_OIDC_ISSUER="$(az aks show -n ${AKS_NAME} -g ${GROUP_NAME} --query "oidcIssuerProfile.issuerUrl" -otsv)"
+```
+
+6. Establish federated identity credential. On AZ CLI `${RATIFY_NAMESPACE}` is where you deploy Ratify:
+
+```
+az identity federated-credential create \
+  --name ratify-federated-credential \
+  --identity-name "${IDENTITY_NAME}" \
+  --resource-group "${GROUP_NAME}" \
+  --issuer "${AKS_OIDC_ISSUER}" \
+  --subject system:serviceaccount:"${RATIFY_NAMESPACE}":"ratify-admin"
```

Comment:
I think the default namespace should be changed to `gatekeeper-system` since Ratify should be installed in the same namespace as gatekeeper and that is the default gatekeeper namespace